sklearn/_build_utils/__init__.py

```diff
@@ -86,7 +86,14 @@ def maybe_cythonize_extensions(top_path, config):
             exc.args += (message,)
             raise
 
+        try:
+            import joblib
+            n_jobs = getattr(joblib, "effective_n_jobs")()
+        except (ImportError, AttributeError):
```

Comment:
why not just `n_jobs = joblib.effective_n_jobs()` ?

Reply:
Joblib 0.11 doesn't have `effective_n_jobs` I believe

Reply:
I mean why do you need to use getattr ? An AttributeError would be raised also by trying to call joblib.effective_n_jobs.

Reply:
Nevermind, it exists. My concern was that prior to joblib 0.13.0 (that included this fix https://github.com/tomMoral/loky/pull/114) was over-estimated in some cases such as CI that has 32 CPU cores, but actually only 2 can be used.

Changed this PR accordingly.